standard_reports/item-usage/latest/convert_data_js/src/test/input.json

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
his file was moved and changed to fully reflect the input data (which I think it should)